scripts/launch-qemu.sh

```diff
@@ -332,38 +332,51 @@ add_opts "-drive if=pflash,format=raw,unit=1,file=${UEFI_BIOS_VARS}"
 # add CDROM if specified
 [ -n "${CDROM_FILE}" ] && add_opts "-drive file=${CDROM_FILE},media=cdrom,index=0"
 
+get_image_format() {
+	IMAGE_FILE=$1
+	if [ ! -f "${IMAGE_FILE}" ]; then
+		echo "${LINENO} Error: Missing argument or incorrect paramter \"$1\" for function ${FUNCNAME}";
```

Comment:
No need to say `{LINENO} Error:` or `for function ${FUNCNAME}`.  We don't do it like that elsewhere, it may confuse users, and developers don't really need it. Also, fix typo in 'paramter'.

Reply:
Sure. I was just giving more information on the error. I can remove it.